plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -1380,7 +1380,7 @@ void ProxyShape::variantSelectionListener(SdfNotice::LayersDidChange const& noti
     return;
   }
 
-  TF_FOR_ALL(itr, notice.GetChangeListMap())
+  TF_FOR_ALL(itr, notice.GetChangeListVec())
```

Comment:
@murphyeoin @robthebloke: Are you still aiming to have the maya-usd dev branch buildable against USD 19.01? Should I wrap this in a USD_VERSION_NUM check? If we don't need to support the old USD anymore, I'd rather not, but no problem adding it if we do. Just let me know!

The UsdShadeMaterial API changes below should be fine, since the "new" API it uses is present in 19.01.

Reply:
I just assumed the answer was going to be yes here and added the version check. :) Easy enough to remove later if/when we find we no longer need it.